ci/jenkins/test.sh

```diff
@@ -519,6 +524,19 @@ function deliver_antrea_windows {
     echo "==== Finish building and delivering Windows images ===="
 }
 
+function copy_kind_test_image {
+    common_images=("registry.k8s.io/e2e-test-images/agnhost:2.40" \
+    "antrea/nginx:1.21.6-alpine" "antrea/systemd-logs:v0.4" \
+    "antrea/toolbox:1.3-0" "antrea/mcjoin:v2.9" \
+    "antrea/antrea-agent-ubuntu:$BUILD_TAG" \
+    "antrea/antrea-controller-ubuntu:$BUILD_TAG" \
+    "antrea/flow-aggregator:$BUILD_TAG")
+
+    for image in "${common_images[@]}"; do
+      kind load docker-image $image --name ${KIND_CLUSTER}
```

Comment:
If some of the images in the list have not been pulled by `docker pull`, will the `kind load` command automatically pull them?

Reply:
no, it will failed with image not found, we can add image check condition before load cmd to pull images if not present on system.

Reply:
Ok, please add it to avoid the potential issue.